### PR TITLE
Adding analyzer comments for Salary Calculator

### PR DESCRIPTION
**Pull request by manumafe98**
Related [#157](https://github.com/exercism/java-analyzer/pull/157)

The goal is to add comments for the new salary calculator analyzer
I also deleted an unnecessary black space on a wizards and warriors comment

@sanderploegsma

### CODE REVIEW

**Comment by manumafe98** [comment]
Just in case, do not merge this one until further notice, because the analyzer was merged by mistake and we are reverting it

File: analyzer-comments/java/salary-calculator/use_ternary_operators.md
Comment (sanderploegsma) [inline analyzer-comments/java/salary-calculator/use_ternary_operators.md:3]: 1. Maybe the copy can be a little bit more verbose and explain where and why a ternary operator should be used. As for where, presumably this comment is triggered when the student uses an `if`/`else` statement, right? Perhaps include that in the copy. As for why: maybe we can point out that the goal of this exercise is to learn about ternary operators and so solutions should use them instead of something else.
3. I would not put the phrase "ternary operators" in backticks, as that is usually only done when referring to Java code or keywords.

**Review by sanderploegsma — CHANGES_REQUESTED** [review]

File: analyzer-comments/java/salary-calculator/use_ternary_operators.md
Comment (manumafe98) [reply analyzer-comments/java/salary-calculator/use_ternary_operators.md]: Done!

**Review by sanderploegsma — APPROVED** [review]

**Comment by manumafe98** [comment]
It can be merged now! when reviewed! thanks

**Review by IsaacG — APPROVED** [review]

**Pull request merged**
